packages/sources/aleno/src/endpoint/price.ts

```diff
@@ -0,0 +1,55 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { TransportRoutes } from '@chainlink/external-adapter-framework/transports'
+import { config } from '../config'
+import overrides from '../config/overrides.json'
+
+import { httpTransport } from '../transport/price-http'
+// import { customSubscriptionTransport } from '../transport/price-custombg'
+
+// // Input parameters define the structure of the request expected by the endpoint. The second parameter defines example input data that will be used in EA readme
```

Comment:
I think those comments make sense during the development, but not sure if we need them in the repo. Applies to all the comments auto generated by ea-generator